libvast/src/system/local_segment_store.cpp

```diff
@@ -36,10 +38,11 @@ namespace {
 // Handler for `vast::query` that is shared between active and passive stores.
 // Precondition: Query type is either `count` or `extract`.
 template <typename Actor>
```

Comment:
What about something like:

```suggestion
// Precondition: Query type is either `count` or `extract`.
/// @returns The matching/extracted rows.
template <typename Actor>
```